Python.md

```diff
@@ -0,0 +1,25 @@
+Python
+======
+Python is a clear and powerful object-oriented programming language, comparable to Perl, Ruby, Scheme, or Java.  Its main focus is on readability and standards based approached
+
+PIP and Environment Management
+---
+Use ``pip`` to install applications and [virtualenv](https://github.com/codeforamerica/howto/blob/master/Python-Virtualenv.md) to manage your development environments.
```

Comment:
Relative `Python-Virtualenv.md` is probably okay for the link, in case this gets forked someplace sometime.